node/src/actors/chain_manager/mod.rs

```diff
@@ -422,7 +422,8 @@ impl ChainManager {
                 if self.sm_state == StateMachine::WaitingConsensus
                     || self.sm_state == StateMachine::Synchronizing
                 {
-                    self.candidates.insert(hash_block, block);
+                    self.candidates.insert(hash_block, block.clone());
+                    self.broadcast_item(InventoryItem::Block(block));
```

Comment:
Could add a comment:
```suggestion
                    // If the node is not synced, broadcast candidates without validating them
                    self.broadcast_item(InventoryItem::Block(block));
```